bazel/external/BUILD

```diff
@@ -4,7 +4,6 @@ cc_library(
     name = "all_external",
     srcs = [":empty.cc"],
     deps = [
-        "@com_github_lightstep_lightstep_tracer_cpp//:lightstep",
```

Comment:
This target is used to pre-build the dependencies when generating the Docker image. We still want to pre-build Lightstep and all its deps since they're pretty big.

Reply:
Fixed.